src/app/public/modules/vertical-tabset/fixtures/vertical-tabset.component.fixture.html

```diff
@@ -1,6 +1,7 @@
 <sky-vertical-tabset
   ariaRole="tablist"
-  showTabsText='Tab list'>
+  showTabsText='Tab list'
+  [maintainTabContent]="maintainTabContent">
```

Comment:
Closing bracket needs to be on a new line.

Reply:
Done